ffi/util.lua

```diff
@@ -60,10 +60,27 @@ int WideCharToMultiByte(
 );
 ]]
 
+local getlibprefix = function()
+    -- Apple M1 homebrew installs libraries outside of default searchpaths,
+    -- and dyld environment variables are sip-protected on MacOS, cf. https://github.com/Homebrew/brew/issues/13481#issuecomment-1181592842
+    local libprefix = os.getenv("KO_DYLD_PREFIX");
```

Comment:
Sorry, I'm a js guy :v Btw I see linter remote checks. Should I get a warning for this locally (in a properly configured editor, that is) based on some definitions in the source?

Reply:
We don't currently have anything that checks for this because it's a style thing and not really an issue otherwise. Also (almost) no one adds redundant semicolons in Lua. :-P